docs/source/getting_started/getting_started.md

```diff
@@ -0,0 +1,226 @@
+
+## Similar documents
+
+In this search variant only a _document id_ must be provided to the [`/semantic_search`](https://docs.xayn.com/front_office.html#tag/front-office/operation/getSimilarDocuments) endpoint.
+
+```bash
+curl -X POST https://<url>/semantic_search
+    -H "authorizationToken: <front_office_token>"
+    -H "Content-Type: application/json"
+    -d '{
+        "document": { "id": "xayn_cd5604c" }
+    }'
+```
+
+The result contains a list of documents that are similar to the `snippet` of the provided _document id_.
+
+## Free Text search
+
+Just like [Similar documents](#similar-documents) it is also possible to run a free text search.
+
+```bash
+curl -X POST https://<url>/semantic_search
+    -H "authorizationToken: <front_office_token>"
+    -H "Content-Type: application/json"
+    -d '{
+        "document": { "query": "Privacy and security" }
+    }'
+```
+
+The quality of the results can vary on the length of the provided query. Short queries usually yield better results with the [hybrid search option](https://docs.xayn.com/front_office.html#tag/front-office/operation/getSimilarDocuments) enabled, that combines semantic and bm25 search:
+
+```json
+{
+  "document": { "query": "Privacy and security" },
+  "enable_hybrid_search": true
+}
+```
+
+## Personalised search
+
+Any search can also be combined with an `user id` or a user history, which is a list of `interactions`.
+To personalise search results for a specific user, any search can also be combined with an `user id` or a user `history`, which is a list of interactions of a particular user. The option to use a user history of interactions instead of a user id enables a personalised search without the need for Xayn to store a user id or history of interactions.
+
+This is how we ask the system for a personalised search result for a [user](#recommendations-personalised-documents):
+
+```bash
+curl -X POST https://<url>/semantic_search
+    -H "authorizationToken: <front_office_token>"
+    -H "Content-Type: application/json"
+    -d '{
+        "document": { "query": "Privacy and security" },
+        "personalize": {
+            "exclude_seen": true,
+            "user": {
+                "id": "u1234",
+            }
+        }
+    }'
+```
+
+The result is a list of documents that match the search query, which are additionally sorted by relevance to the user's interests based on their content interactions.
+
+Alternatively a history of interactions can be used instead of a user id to ask for personalised documents:
+
+```json
+"personalize": {
+    "exclude_seen": true,
+    "user": {
+       "history": [
+            {
+                "id": "valid_doc_id1",
+                "timestamp": "2000-05-14T20:22:50Z"
+            },
+            {
+                "id": "valid_doc_id2",
+                "timestamp": "2000-05-15T20:22:50Z"
+            }
+        ]
+    }
+}
+```
```

Comment:
I would add here some like:
For details on how to correctly structure the history, please look at: <link>